### PR TITLE
Task header

### PR DESCRIPTION
**Pull request by danicatt**
Created top navbar for Portfolio.
Created header section for portfolio - background image and overlapping text

### CODE REVIEW

File: task-header.css [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+	}
+
+/* Navbar */
+
+.firstnav {
+	display: flex;
+	flex-direction:row;
+	justify-content: space-around;
+	width:100%;
+	height: 75px;
```
Comment (alexjennings56) [inline task-header.css:17]: inconsistent spacing, just for tidiness :) 
```suggestion
	height:75px;
```

File: task-header.css [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+	}
+	
+
+.socialicons {
+	width: 35px;
+	height: 35px;
+	background-color: rgb(191, 217, 218);
+}
+
+
```
Comment (alexjennings56) [inline task-header.css]: Consider tightening up gaps between classes?

File: task-header.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<html>
+	<head>
+		<link rel="stylesheet" href="task-header.css">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+	</head>
+	
+	<body>
+		<nav class="firstnav">
+			<div class="infobar">
+				<a href="#"> About Me </a>
```
Comment (alexjennings56) [inline task-header.html]: Might be worth adding names to the # just for practice.
```suggestion
				<a href="#aboutme"> About Me </a>
```

File: task-header.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+				<a href="https://twitter.com/?lang=en-gb"><img src="twitter.png" img class="socialicons"></a>
+				<a href="https://www.instagram.com/"><img src="instagram.png" img class="socialicons"></a>
+			</div>
+		</nav>
+		
+		<header class="mainintro">
+			<div class="maintitle">
+				<h1>Danielle Catterill</h1>
+				<h2>Tea Drinker, Dog Lover & Full Stack Developer in Training</h2>
+			<div>
```
Comment (alexjennings56) [inline task-header.html:26]: div not closed 
```suggestion
			</div>	
```

**Review by alexjennings56 — CHANGES_REQUESTED** [review]

File: task-header.css (hunk 1 above)
Comment (danicatt) [reply task-header.css:17]: Good spot! I've updated my code so there is now a single space.

File: task-header.css (hunk 2 above)
Comment (danicatt) [reply task-header.css]: I've gone through and closed the gaps. Should now just be a single gap between classes.

File: task-header.html (hunk 3 above)
Comment (danicatt) [reply task-header.html]: I was going to leave them # until I played around with adding links. But see the value in stating what the href's are.

File: task-header.html (hunk 4 above)
Comment (danicatt) [reply task-header.html:26]: Thanks :)

**Review by danicatt — COMMENTED** [review]
Just replying to your comments

**Review by alexjennings56 — APPROVED** [review]

**Pull request merged**